repository/definitions/vulnerability/oval_com.gfi_def_149.xml

```diff
@@ -0,0 +1,134 @@
+<definition xmlns="http://oval.mitre.org/XMLSchema/oval-definitions-5" class="vulnerability" id="oval:com.gfi:def:149" version="0">
+    <metadata>
+        <title>Internet Explorer Memory Corruption Vulnerability</title>
+        <affected family="windows">
+            <platform>Microsoft Windows 7</platform>
+            <platform>Microsoft Windows 8.1</platform>
+            <platform>Microsoft Windows 10</platform>
+            <platform>Microsoft Windows 10 Version 1607</platform>
```

Comment:
Same comment, I would suggest removing the Version-specific <platform>s and just leaving Microsoft Windows 10.